logstash-core-event/spec/logstash/timestamp_spec.rb

```diff
@@ -81,4 +81,29 @@
       expect(t - 10).to eq(current)
     end
   end
+
+  context "identity methods" do
+    subject { LogStash::Timestamp.new }
+
+    it "should support utc" do
+      expect(subject.utc).to eq(subject)
```

Comment:
As I understand the ruby API, calling UTC converters this value, right? I guess we have test for this already, if so I would recommend grouping this test together for clear understanding.